pyproject.toml

```diff
@@ -39,18 +39,17 @@ poethepoet = "^0.24.4"
 coverage = "^7.2.1"
 pytest-asyncio = "^0.21.1"
 pylint = "3.0.3"
-setuptools = "^69.0.3"
 pytest-mock = "^3.6.1"
 pytest-xdist = "^3.2.1"
 pyright = "1.1.338"
 pytest-cov = "^4.0.0"
 isort = "^5.11.4"
 pytest-rerunfailures = "^13.0"
 python-dotenv = "^1.0.0"
+setuptools = "^69.0.3"
 
-
-[tool.poetry.group.py39-dev.dependencies]
-cairo-lang = {version = "0.13.0", python = ">=3.9, <3.10"}
+[tool.poetry.group.py312-dev.dependencies]
+cairo-lang = {version = "0.13.0", python = ">=3.9, <3.13"}
```

Comment:
Please shift this to the `dev.dependencies` section, omit the Python requirement, and accordingly adjust the workflow.

Reply:
It doesnt works :( Maybe I did sth wrong could you take a look?